src/containers/App/App.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { func } from 'prop-types';
 import { translate } from 'react-i18next';
-import 'i18n/i18n';
+import '../../i18n/i18n';
 import styles from './assets/app.scss';
+import Contact from '../../components/contact/Contact';
```

Comment:
You can just import a component like this: `import Contact from 'components/contact/Contact';`